src/main/scala/com/databricks/labs/overwatch/utils/UpgradeExecutor.scala

```diff
@@ -0,0 +1,72 @@
+package com.databricks.labs.overwatch.utils
+
+import com.databricks.labs.overwatch.env.Workspace
+import org.apache.spark.sql.DataFrame
+
+import scala.collection.mutable
+
+class UpgradeExecutor(val etldbname: String, val fromVersion: Int, val toVersion: Int, val executeSingleStep: Int = -1) {
+
+  val versionMap = Map("600-609" -> classOf[UpgradeTo0610], "610-699" -> classOf[UpgradeTo0700])
```

Comment:
Would you have a versionMap for each upgrade path? Is this just an example for 1 of them?